README.md

```diff
@@ -1,15 +1,15 @@
-# h2
-
-Currently works for youtube and pdf files, Copy any youtube or PDF url and press Ctrl/Command + Shift + V
-
+# H2
 Inspired from Helium app. 
-
+Currently works for youtube and pdf files, Copy any youtube or PDF url and press Ctrl/Command + Shift + V
 [WIP] many features coming soon. Works for all platforms, thanks to electron ❤️
 
-##Bug
-Sometimes app doesn't run over the top of mac screens
-
-
+![sample](img.png)
+## Installation
+## Usage
+1. Can run youtube
+2. Can run pdf and google docs file
+3. After [#22](https://github.com/poush/H2/pull/22) it will have a basic image editor, so we can include this as well.
+## Contributing
```

Comment:
We have guides for complete installation steps at `guides/[os].md`. Can you link them here instead of these steps? We even bash scripts for Linux and Mac OS

Reply:
These are in the usage area. I can link the os.md files in the installation area. Is that okay? :D